src/DynamoCoreWpf/UI/Prompts/PresetPrompt.xaml.cs

```diff
@@ -18,10 +19,15 @@ public PresetPrompt()
 
             this.Owner = WpfUtilities.FindUpVisualTree<DynamoView>(this);
             this.WindowStartupLocation = WindowStartupLocation.CenterOwner;
-
+            this.Loaded +=PresetPrompt_Loaded;
             this.nameBox.Focus();
         }
 
+        private void PresetPrompt_Loaded(object sender, RoutedEventArgs e)
+        {
+            this.TextRemaining.Content = Wpf.Properties.Resources.PresetTextRemaining + " " + (this.nameBox.MaxLength - this.nameBox.Text.Length);
```

Comment:
Any reason to not turn this into a separate method?  This will make it easier to update this code...